nodelet/pkg/pf9kube/pf9/pf9-kube/phases/prepare_kube_configs.sh

```diff
@@ -25,6 +26,10 @@ function start() {
     if [ "$ROLE" == "master" ]; then
         prepare_rolebindings
     fi
+
+    # Make the config directory read write executable
+    chmod -R 0600 $config_dir
+    chmod 0700 $config_dir
```

Comment:
The execute permissions for the dir was added for hostagent extensions that CD into the dir and then execute. Is that required for edge cloud? I could not find any hostagent extensions directory in the nodelet repo.